.golangci.yml

```diff
@@ -3,7 +3,6 @@ linters:
   enable:
     - deadcode
     - gocyclo
-    - golint
```

Comment:
from https://golangci-lint.run/usage/linters/, it says golint is replaced by `revive`, can we enable `revive` here?

also, can we ref https://github.com/authgear/authgear-server/blob/master/.golangci.yml and add more linters?